docs/app/pages/css/css-sections/icons/ux-icons/icon-preview/icon-preview.component.ts

```diff
@@ -0,0 +1,63 @@
+import { ChangeDetectionStrategy, Component, ElementRef, Input, Renderer2, ViewChild } from '@angular/core';
+import { TooltipDirective } from '@ux-aspects/ux-aspects';
+
+@Component({
+    selector: 'uxd-icon-preview',
+    templateUrl: './icon-preview.component.html',
+    styleUrls: ['./icon-preview.component.less'],
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class IconPreviewComponent {
+    @Input() name: string;
+    @Input() classname: string;
+    @Input() iconset: string;
+    @Input() iconClass: string;
+
+    public tooltipText: string = 'Click to copy icon text';
```

Comment:
Could we remove the `public` keyword, by default properties are public so we normally just omit this.

Reply:
Removed